src/droplet.F

```diff
@@ -1539,7 +1550,7 @@ end subroutine interpolate_to_parcel
       subroutine find_horizontal_location_index (loc_ind, loc, lb, ub, dsize, end_ind, ind)
       !$acc routine seq
 
-      use input
+      use input, only :
```

Comment:
It seems that we do not need to use the `input` module here.

Reply:
Removed the use input statement.